### PR TITLE
FB link added

### PR DESCRIPTION
**Pull request by ghost**
I did add the WWCode Cincy FB link.

### CODE REVIEW

**Comment by sierraobryan** [comment]
@a1cha can you go ahead and add the rest of the links in this PR so they are ready to go and we can use them in the projects 😅  please follow the naming technique in the links object example structure inside the `member` example in issue #1 
I also tagged you on issue #4 which includes adding all of the contact information

File: db.json [hunk 1]
```diff
@@ -72,7 +72,8 @@
 	}],
 	"women_who_code_cincinnati_information": {
 		"links": {
-			"twitter": "https://twitter.com/CincyWWCode"
+			"twitter": "https://twitter.com/CincyWWCode",
+			"facebook": "https://www.facebook.com/groups/wwcodecincinnati/"
```
Comment (sierraobryan) [inline db.json]: Please add all information: website (official https://www.womenwhocode.com/cincinnati), meetup, and github following the expected JSON structure 
```
ContactInformation {
  "links": Links,
  "email": String 
}
```

**Comment by ghost** [comment]
Hey Sierra,

Sure. I will. I added only this FB one link as a follow of resolving an issue in Gatsby, and it relied on the database, so I wanted it to work.
I still try to figure our the flutter app, haha.
Have a good day!
A.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by sierraobryan** [comment]
Thank @a1cha - please add and commit the other links when you have a chance and we will merge this PR!

**Pull request merged**